src/benchmarks/real-world/Roslyn/RoslynApis.cs

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
I am sorry, I forgot to include new using statement in my previous GH suggestion which breaks the CI.

```suggestion
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.

using System.Collections.Immutable;
```